showcase/echo_test.go

```diff
@@ -68,19 +69,25 @@ func TestEcho_error(t *testing.T) {
 		},
 	}
 	for typ, client := range map[string]*showcase.EchoClient{"grpc": echo, "rest": echoREST} {
-		if typ == "rest" {
-			// TODO(dovs): currently erroring with 2, want 1
-			continue
-		}
-
 		resp, err := client.Echo(context.Background(), req)
-		if resp != nil {
+		if resp != nil || err == nil {
 			t.Errorf("%s Echo() = %v, wanted error %d", typ, resp, val)
 		}
-		status, _ := status.FromError(err)
-		if status.Code() != val {
-			t.Errorf("%s Echo() errors with %d, want %d", typ, status.Code(), val)
+		if typ == "grpc" {
+			status, _ := status.FromError(err)
+			if status.Code() != val {
+				t.Errorf("%s Echo() errors with %d, want %d", typ, status.Code(), val)
+			}
+		} else {
+			const ERR_CODE int = 500 // Unavailable
```

Comment:
Use `StatusInternalServerError` from https://pkg.go.dev/net/http instead please. also 503 is unavailable, 500 is internal server error.

Reply:
That is exactly what I was looking for. Thanks!